tests/unit/QuantFile.spec.js

```diff
@@ -356,32 +356,100 @@ describe('QuantFile.vue', () => {
     })
   })
 
-  describe('custom plate sizes', () => {
+  describe('Heron TapeStation Tubes', () => {
```

Comment:
I did this specifically as we should be testing the behaviour of QuantFile here, not our configuration. I realise most the other tests are coupled to configuration, so perhaps consistency is better.